docs/manual/index.md

```diff
@@ -60,7 +60,7 @@ feedback](/mojo/community).
 
 - **Python**
 
-  - [Python integration](/mojo/manual/python/index)
+  - [Python integration](/mojo/manual/python)
```

Comment:
```suggestion
  - [Python integration](/mojo/manual/python/)
```

Reply:
We should retain the trailing slash here. Our site appears to accept both, but this is consistent with the link in the left nav.